### ISSUE TITLE
Default User Home Personal Library creation in Linux is still prefixed 3.3 for MRO 3.4.0/3.4.1/3.4.2

### ISSUE THREAD

**Issue by mjmg** [Microsoft/microsoft-r-open#27]
On Linux , the default library install location proposed/used is still on the subdirectory named "3.3"

Tested under both RPM and Debian based environments.

```
> install.packages("packagename")                                                                                                
Warning in install.packages("packagename") :
  'lib = "/usr/lib64/microsoft-r/3.4/lib64/R/library"' is not writable
Would you like to use a personal library instead?  (y/n) y
Would you like to create a personal library
~/R/x86_64-pc-linux-gnu-library/3.3
to install packages into?  (y/n) 
```

For Windows, it is correctly on a "3.4" subdirectory.
```
~/R/win-library/3.4
```

**Comment by mjmg** [comment]
This issue still exists for MRO 3.4.1 under Linux.

**Comment by mjmg** [comment]
This issue still exists for MRO 3.4.2 under Linux. I will be surely disappointed if this gets carried over to MRO 3.5.X

**Comment by mjmg** [comment]
Seems to be fixed in MRO 3.4.3 under Linux

**Comment by richcalaway** [comment]
Yes, this was fixed in MRO 3.4.3.
